src/test/java/org/broadinstitute/hellbender/JBWAIntegrationTest.java

```diff
@@ -54,4 +57,45 @@ public void testJBWAAlignSingleRead() throws Exception {
         Assert.assertEquals(align[0].getNm(), 0);
         bwaMem.dispose();
```

Comment:
I didn't write this test but I've added a try/finally here that disposes the mem and closes the index. Some tests don't need it so I didn't move stuff to before / after class methods.